pandas/core/window.py

```diff
@@ -204,8 +204,21 @@ def __repr__(self):
         )
 
     def __iter__(self):
-        url = "https://github.com/pandas-dev/pandas/issues/11704"
-        raise NotImplementedError("See issue #11704 {url}".format(url=url))
+        self.validate()
```

Comment:
`self.validate()` is already done in the `__init__`

Reply:
Well spotted! Will remove :)